android/src/main/java/com/imagepicker/ImagePickerModule.java

```diff
@@ -347,7 +347,7 @@ public void launchImageLibrary(final ReadableMap options, final Callback callbac
 
     try
     {
-      currentActivity.startActivityForResult(libraryIntent, requestCode);
+      currentActivity.startActivityForResult(Intent.createChooser(libraryIntent, "Choose gallery app"), requestCode);
```

Comment:
Change "Choose gallery app" to come from the options instead and then I can merge this.